ironic/drivers/modules/drac/management.py

```diff
@@ -389,3 +402,68 @@ def known_good_state(self, task):
         client = drac_common.get_drac_client(node)
         client.reset_idrac(force=True, wait=True)
         client.delete_jobs(job_ids=[_CLEAR_JOB_IDS])
+
+    @METRICS.timer('DracManagement.clear_job_queue')
+    @base.clean_step(priority=0)
+    def clear_job_queue(self, task):
+        """Clear the job queue.
+
+        :param task: a TaskManager instance containing the node to act on.
+        :returns: None if it is completed.
+        :raises: DracOperationError on an error from python-dracclient.
+        """
+        node = task.node
+
+        client = drac_common.get_drac_client(node)
+        client.delete_jobs(job_ids=[_CLEAR_JOB_IDS])
```

Comment:
Might want to pass no args in the line above for simplicity.

Reply:
I think, here we need to keep args as is, because in case of other pending jobs , need this args to clear those pending jobs.

Reply:
Ah, I see.  This is passing in the FORCE version of the parameter, where the default does not.  Let's keep this as-is.